ambry-rest/src/test/java/com.github.ambry.rest/NettyServerTest.java

```diff
@@ -66,32 +67,20 @@ public void shutdownWithoutStartTest() throws InstantiationException, IOExceptio
    */
   @Test
   public void startWithBadInputTest() throws InstantiationException, IOException {
-    Properties properties = new Properties();
-    // Should be int. So will throw at instantiation.
-    properties.setProperty("netty.server.port", "abcd");
-    NioServer nioServer = null;
-    try {
-      nioServer = getNettyServer(properties);
-      fail("NettyServer instantiation should have failed because of bad nettyServerPort value");
-    } catch (NumberFormatException e) {
-      // nothing to do. expected.
-    } finally {
-      if (nioServer != null) {
-        nioServer.shutdown();
-      }
-    }
-
-    // Should be > 0. So will throw at start().
-    properties.setProperty("netty.server.port", "-1");
-    nioServer = getNettyServer(properties);
-    try {
-      nioServer.start();
-      fail("NettyServer start() should have failed because of bad nettyServerPort value");
-    } catch (IllegalArgumentException e) {
-      // nothing to do. expected.
-    } finally {
-      if (nioServer != null) {
-        nioServer.shutdown();
+    for (int i = 0; i < 2; i++) {
```

Comment:
minor: would prefer writing a function like `verifyStartFailure(Properties properties)` instead of manipulating using loop variables.